.github/workflows/automated-tests.yml

```diff
@@ -0,0 +1,162 @@
+name: Automated Tests
+on:
+  workflow_dispatch:
+
+env:
+  EDITOR_DIRECTORY: editor.planx.uk
+  PNPM_VERSION: 7.8.0
+  NODE_VERSION: 16.13.1
+
+jobs:
+  integration_tests:
+    name: Run Integration tests
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.PIZZA_AWS_ACCESS_KEY_ID }}
```

Comment:
will these `PIZZA_` variables actually work if this is triggered on the `main` branch do you think? Just thinking about recent scenarios of broken Notify templates where it'd be convenient to check/compare state of `main`, not a local PR branch? 

testing actions is tricky though so happy for this to merge & fix forward if needed :slightly_smiling_face:

Reply:
These aren't specific to a Vultr deployment but general AWS keys. I was aiming to change as little as possible to make future de-duping easier later and I _think_ this should work :crossed_fingers: